src/registrar/models/user.py

```diff
@@ -163,6 +163,16 @@ def get_active_requests_count(self):
         active_requests_count = self.domain_requests_created.filter(status__in=allowed_states).count()
         return active_requests_count
 
+    def get_expiring_domains(self, request):
```

Comment:
I would suggest either:
- if knowing which domains are expiring isn't important -- renaming this to get_num_expiring_domains
- if knowing which domains are expiring is important and we need to reference it elsewhere -- refactoring this method so that we save which domains are expiring and referencing it by len(expiring_domains)

Reply:
@erinysong I can update the name to `get_num_expiring_domains`, and removed the feature flag check.